src/catsDb.js

```diff
@@ -1,34 +1,38 @@
-// class 'CatRepository' serves as a template for creating new cat objects with props & methods
+// class 'CatClass' serves as a template for creating new cat objects with props & methods
+// example of dependancy injection:
 
-class CatRepository {
-    constructor() {
-        this.cats = [ // array of objects - if object, change to >> this.cats:{ 1:{}, 2:{}, 3:{}}
-            {
-                id: 1,
+class CatClass {
```

Comment:
The requirements say that this should be named `CatRepository`. This is a more specific and descriptive name - if you were looking at this codebase for the first time you wouldn't know what a `CatClass` is, but you'd have an inkling of what a repository is and what it does.

Reply:
Done.

Reply:
This can just be named `CatRepository` - adding class onto the end is nearly always omitted as it's possible to get that information from the context. Although I can appreciate this naming might help a little when starting out it'd be best to stick with convention

Reply:
Its the distinguish easier the class from the class instance, but yes ok.